core/client/batch_get_and_verify.go

```diff
@@ -30,13 +33,57 @@ func (c *Client) BatchVerifyGetUserIndex(ctx context.Context, userIDs []string)
 		return nil, err
 	}
 
+	_, spanEnd := monitoring.StartSpan(ctx, "VerifyBatchGetUserIndex")
```

Comment:
Did you intend this to match method name? OTOH, it doesn't span the whole method, so maybe names should be even more different.

Reply:
Why not both? `BatchVerifyGetUserIndex.Verify`